selfdrive/car/hyundai/interface.py

```diff
@@ -181,12 +181,12 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
     elif candidate == CAR.KIA_EV6:
       ret.mass = 2055 + STD_CARGO_KG
       ret.wheelbase = 2.9
-      ret.steerRatio = 16.
+      ret.steerRatio = 14.25 # https://www.kiamedia.com/us/en/models/ev6/2022/specifications
```

Comment:
```suggestion
      ret.steerRatio = 14.25  # https://www.kiamedia.com/us/en/models/ev6/2022/specifications
```

our ev6 learned 14.85, pretty close